base/actions/actions.py

```diff
@@ -38,6 +38,10 @@ def yes_no_data(self) -> Dict[int, Text]:
     def yes_no_maybe_data(self) -> Dict[int, Text]:
         return {1: "yes", 2: "no", 3: "not sure"}
 
+    @property
+    def yes_no_more_data(self) -> Dict[int, Text]:
```

Comment:
Do we need this? We never save "more" or validate against it, we redirect before really validating

Reply:
cool